docs/example_of_generating_movie.rst

```diff
@@ -0,0 +1,137 @@
+.. _example_of_generating_movie:
```

Comment:
can you try putting the ipynb from google colab here and building with `make docs` instead of `make sphinx` to see if that works? Alternatively, can you add the video?

Reply:
should the make docs be taking this long? let me know, because its speed is more a couple of minutes than seconds with make sphinx